acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/ValidatorKeysAcceptanceTest.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.test.acceptance;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Iterator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import tech.pegasys.teku.bls.BLSPublicKey;
+import tech.pegasys.teku.provider.JsonProvider;
+import tech.pegasys.teku.test.acceptance.dsl.AcceptanceTestBase;
+import tech.pegasys.teku.test.acceptance.dsl.BesuNode;
+import tech.pegasys.teku.test.acceptance.dsl.TekuNode;
+import tech.pegasys.teku.test.acceptance.dsl.TekuValidatorNode;
+import tech.pegasys.teku.test.acceptance.dsl.tools.deposits.ValidatorKeystores;
+
+public class ValidatorKeysAcceptanceTest extends AcceptanceTestBase {
+  private final JsonProvider jsonProvider = new JsonProvider();
+
+  @Test
+  void shouldMaintainValidatorsInMutableClient(@TempDir final Path tempDir) throws Exception {
+    final String networkName = "less-swift";
+    final BesuNode eth1Node = createBesuNode();
+    eth1Node.start();
+
+    final ValidatorKeystores validatorKeystores =
+        createTekuDepositSender(networkName).sendValidatorDeposits(eth1Node, 8);
+    final ValidatorKeystores extraKeys =
+        createTekuDepositSender(networkName).sendValidatorDeposits(eth1Node, 1);
+
+    final TekuNode beaconNode =
+        createTekuNode(config -> config.withNetwork(networkName).withDepositsFrom(eth1Node));
+
+    final TekuValidatorNode validatorClient =
+        createValidatorNode(
+                config ->
+                    config
+                        .withNetwork(networkName)
+                        .withValidatorApiEnabled()
+                        .withInteropModeDisabled()
+                        .withBeaconNode(beaconNode))
+            .withMutableValidatorKeystores(validatorKeystores);
```

Comment:
I'm unclear why we're passing in `validatorKeystores` here and then expecting them to not be active.  I'd have though this would just be something like `.enableValidatorApi()`.

Reply:
i was initially going to load them, but then decided I wanted it to initially be empty, and didnt circle back - cleaned up.